### PR TITLE
Add Fuxi Volume Plugin to the docs

### PR DESCRIPTION
**Pull request by hongbin**
Signed-off-by: Hongbin Lu <hongbin034@gmail.com>

### CODE REVIEW

File: docs/extend/legacy_plugins.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Plugin
 [REX-Ray plugin](https://github.com/emccode/rexray)                                 | A volume plugin which is written in Go and provides advanced storage functionality for many platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 [Virtuozzo Storage and Ploop plugin](https://github.com/virtuozzo/docker-volume-ploop) | A volume plugin with support for Virtuozzo Storage distributed cloud file system as well as ploop devices.
 [VMware vSphere Storage Plugin](https://github.com/vmware/docker-volume-vsphere)    | Docker Volume Driver for vSphere enables customers to address persistent storage requirements for Docker containers in vSphere environments.
+[Fuxi Volume Plugin](https://github.com/openstack/fuxi)                             | A volume plugin that is developed as part of the OpenStack Kuryr project and implements the Docker volume plugin API by utilizing Cinder, the OpenStack block storage service.
```
Comment (thaJeztah) [inline docs/extend/legacy_plugins.md]: Can you move the plugin "up", so that the list is in alphabetical order?

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Comment by hongbin** [comment]
@thaJeztah , thanks for the review. I have updated the pull request to address your comments.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

/cc @vdemeester

**Review by vdemeester — APPROVED** [review]
LGTM :frog:

**Pull request merged**